From bff35147066a2aaeff0c5f7cce80a298834f52f5 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Sat, 09 Jul 2022 05:31:33 +0300
Subject: [PATCH] Revert "[SW-104491] rom: tpm: store pcr in sha256 bank"

This reverts commit a3e0f2b2b17efe3f08716e6ccd314af5a5c54510.

As per the updated security specification sha384 bank is to be used
to extend the tpm pcr. Hence the intermediate decisions to change
the bank is reverted.

Change-Id: I22f6866595093b0c336a4b7f55128b38f816da30
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 4890352..a3a9fd7 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -840,7 +840,7 @@
 	if (HL_FW_IMG_NOT_FOUND == idx)
 		return 1;
 
-	ret = tpm2_pcr_extend(TPM2_PPBOOT_PCR_IDX, TPM2_ALG_SHA256, desc.chksum, SHA384_SUM_LEN);
+	ret = tpm2_pcr_extend(TPM2_PPBOOT_PCR_IDX, TPM2_ALG_SHA384, desc.chksum, SHA384_SUM_LEN);
 	return ret;
 }
 
